pkg/limayaml/defaults.go

```diff
@@ -699,6 +699,12 @@ func FillDefault(y, d, o *LimaYAML, filePath string, warn bool) {
 	for _, f := range y.MountTypesUnsupported {
 		mountTypesUnsupported[f] = struct{}{}
 	}
+
+	if runtime.GOOS == "windows" {
+		// QEMU for Windows does not support 9p
```

Comment:
Should this be mentioned in a comment in `default.yaml`?

Reply:
done